backend/build.gradle

```diff
@@ -64,9 +64,8 @@ dependencies {
 
     // Okta dependencies
     implementation 'com.okta.spring:okta-spring-boot-starter:2.1.7'
-    implementation 'com.okta.sdk:okta-sdk-api:8.2.1'
-    runtimeOnly 'com.okta.sdk:okta-sdk-impl:8.2.1'
-    runtimeOnly 'com.okta.sdk:okta-sdk-httpclient:8.2.1'
+    implementation 'com.okta.sdk:okta-sdk-api:12.0.1'
+    runtimeOnly 'com.okta.sdk:okta-sdk-impl:12.0.1'
```

Comment:
Looks like version `12.0.2` for the sdk is out with some very [minor changes](https://github.com/okta/okta-sdk-java/releases/tag/okta-sdk-root-12.0.2), wondering if its worth the retest effort to upgrade to it